src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -94,6 +94,46 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task HttpHeader_LastModifiedDate_Success()
```

Comment:
The test names here should be something like this:

"LastModified_ValidDate_Success"
"LastModified_InvalidDate_Throws"

This follows the pattern of <property or method under test>_<conditions>_<expected result>